transforms/universal/fdedup/kfp_ray/fdedup_wf.py

```diff
@@ -204,21 +242,147 @@ def fdedup(
+            duplicate_list_location=fdedup_duplicate_list_location,
+        )
+        ComponentUtils.add_settings_to_component(compute_get_duplicate_list_exec_params, ONE_HOUR_SEC * 2)
+        compute_get_duplicate_list_exec_params.after(execute_cluster_analysis_job)
+        # Execute job
+        execute_get_duplicate_list_job = execute_get_duplicate_list_job_op(
+            ray_name=ray_name,
+            run_id=run_id,
+            additional_params=additional_params,
+            exec_params=compute_get_duplicate_list_exec_params.output,
+            exec_script_name=GET_DUPLICATE_LIST_EXEC_SCRIPT_NAME,
+            server_url=server_url,
+        )
+        ComponentUtils.add_settings_to_component(execute_get_duplicate_list_job, ONE_WEEK_SEC)
+        # FIXME: see https://github.com/kubeflow/pipelines/issues/10914
+        if os.getenv("KFPv2", "0") != "1":
+            ComponentUtils.set_s3_env_vars_to_component(execute_get_duplicate_list_job, data_s3_access_secret)
+        execute_get_duplicate_list_job.after(compute_get_duplicate_list_exec_params)
+
+        compute_data_cleaning_exec_params = compute_data_cleaning_exec_params_op(
+            runtime_actor_cpus=runtime_actor_cpus,
+            runtime_num_actors=runtime_num_actors,
+            data_s3_config=data_s3_config,
+            data_max_files=data_max_files,
+            data_num_samples=data_num_samples,
+            runtime_pipeline_id=runtime_pipeline_id,
+            runtime_job_id=run_id,
+            runtime_code_location=runtime_code_location,
+            id_column=fdedup_document_id_column,
+            duplicate_list_location=fdedup_duplicate_list_location,
+            operation_mode=fdedup_operation_mode,
+        )
+        ComponentUtils.add_settings_to_component(compute_data_cleaning_exec_params, ONE_HOUR_SEC * 2)
+        compute_data_cleaning_exec_params.after(execute_get_duplicate_list_job)
+
         # Execute job
-        execute_job = execute_ray_jobs_op(
+        execute_data_cleaning_job = execute_data_cleaning_job_op(
             ray_name=ray_name,
             run_id=run_id,
             additional_params=additional_params,
-            exec_params=compute_exec_params.output,
-            exec_script_name=EXEC_SCRIPT_NAME,
+            exec_params=compute_data_cleaning_exec_params.output,
+            exec_script_name=DATA_CLEANING_EXEC_SCRIPT_NAME,
             server_url=server_url,
+            prefix="dcdata",
         )
-        ComponentUtils.add_settings_to_component(execute_job, ONE_WEEK_SEC)
-        ComponentUtils.set_s3_env_vars_to_component(execute_job, data_s3_access_secret)
-        execute_job.after(ray_cluster)
+        ComponentUtils.add_settings_to_component(execute_data_cleaning_job, ONE_WEEK_SEC)
+        # FIXME: see https://github.com/kubeflow/pipelines/issues/10914
+        if os.getenv("KFPv2", "0") != "1":
+            ComponentUtils.set_s3_env_vars_to_component(execute_data_cleaning_job, data_s3_access_secret)
```

Comment:
As discussed in KFP v2 the secret name is hard coded (in kfp_v2_workflow_support/src/workflow_support/compile_utils/component.py) as a workaround for  https://github.com/kubeflow/pipe lines/issues/10914.  Thus this pipeline is not expected to run on kfp V2. Should it be added to black list as we do not have a way to run it only for v1 in ci/cd? @roytman what do you think? Thanks